common/src/main/java/org/jboss/gm/common/Configuration.java

```diff
@@ -73,6 +73,19 @@ default Properties getProperties() {
     @Key("accessToken")
     String accessToken();
 
+    @Key("versionOsgi")
+    @DefaultValue("true")
+    boolean versionOsgi();
+
+    @Key("versionOverride")
+    String versionOverride();
+
+    @Key("versionSuffix")
+    String versionSuffix();
+
+    @Key("versionSuffixAlternatives")
+    String versionSuffixAlternatives();
+
```

Comment:
I am in two minds about these. Technically they override what can be picked up already within the PME code. However saying that, it does make it explicit which is easier to follow for maintenance. Perhaps adding a comment describing that these are merely here to make it explicit?